ORStools/proc/isochrones_layer_proc.py

```diff
@@ -122,20 +122,11 @@ def processAlgorithm(self, parameters, context, feedback):
 
         # Get ID field properties
         id_field_name = parameters[self.IN_FIELD]
-        id_field_id = source.fields().indexOf(id_field_name)
+        if id_field_name:
+            id_field = source.fields().field(id_field_name)
+            parameter_options = [id_field.type(), id_field_name]
```

Comment:
referenced before assignment for `id_field_name` = false

Reply:
done